<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MS data analysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_file_action"/>
   <addaction name="separator"/>
   <addaction name="actionPeaks"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="xyTable">
   <property name="windowTitle">
    <string>Peak table</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="xyTableContents">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="xyTableLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="open_file_action">
   <property name="icon">
    <iconset>
     <normalon>:/Icons/file_open_icon</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open data file</string>
   </property>
   <property name="toolTip">
    <string>Loads ascii delimited data from a file</string>
   </property>
  </action>
  <action name="actionPeaks">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/peak</normaloff>:/Icons/peak</iconset>
   </property>
   <property name="text">
    <string>Peaks calc.</string>
   </property>
   <property name="toolTip">
    <string>Calculates peak positions</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
